### DISCUSSION TITLE
video stream esp32cam - Tasmota firmware 12.5.0.1

### DISCUSSION THREAD

**Discussion by zlucio** [arendst/Tasmota#18571] [Q&A]
I'm looking for help and answers. !! On 2 esp32cam AI-Thinker cam modules I upgraded to version 12.5.0.1 and the video stream (http://192.168.8.85:81/stream) stops after a few seconds. I downgraded it from 11.1.0 and it works.
Thank you

**Comment by Jason2866** [comment]
The AI Thinker cam and clones are know to be hardware design crap. There are better alternatives. Cam Boards based on ESP32-S3 are working fine, with EVERY Tasmota version.
My AI Thinker Cam has issues with every software i tried. After a few hours it starts working. And yes with latest Tasmota version too.

See https://github.com/arendst/Tasmota/discussions/18492 https://github.com/arendst/Tasmota/discussions/18482 and more...
